app/scripts/helpers/StaticHelpers.js

```diff
@@ -52,6 +52,14 @@ define(function(){
 
         capitalizeFirstLetter: function(string) {
             return string.charAt(0).toUpperCase() + string.slice(1);
+        },
+
+        createFace: function(data){
+            return new THREE.Face3( data[0], data[1], data[2],
+                [ new THREE.Vector3( data[3][0], data[3][1], data[3][2] ),
+                  new THREE.Vector3( data[3][3], data[3][4], data[3][5] ),
+                  new THREE.Vector3( data[3][6], data[3][7], data[3][8] )
+                ]);
```

Comment:
As much as I like code re-use, it's a bad idea to put such a dependency in a static class.

What if we remove Three as a dependency?  It's not a good idea (especially in a dynamic language like JavaScript) to have a dependency hanging around like this.

If you want to do something like this, I'd prefer that you create a class that explicitly defines the dependency (e.g. ThreeUtilities).  That will be easier to find and have little cost in the minified code.